trusted-committer/es/outline-es.md

```diff
@@ -0,0 +1,87 @@
+# Esquema de la ruta de aprendizaje sobre el Trusted Committer
+
+## Metas de aprendizaje
+
+- Definición de TC (por qué éste nombre)
+- Comprender las responsabilidades del Trusted Committer (TC)``
+- Diferentes maneras de llegar a ser TC
+- Ciclo de vida del TC (preparar sucesores, retirarse)
+- Habilidades útiles/actitud/madurez emocional
+- Antipatrones del TC
+- Por qué es importante el rol del TC para el éxito de IS
+
+## Audiencia Objetivo
+
+- TCs actuales
+- Gente que trabaja/trata con TCs 
+    - Contribuidores
+    - Futuros TCs en potencia
+    - Gestores de TCs
+
+## Esquema
+
+### Definición
+
+- Glosario
+    - Contribuidor/Invitado
+        - Responsable de "encajar en"/cumplir con el equipo anfitrión y sus
+          expectativas/procesos.
+    - Product Owner (distinción entre PO's invitado y anfitrión)
+    - TC
+        - El TC está autorizado a acercar más el código a productión que un 
+          contribuidor
+        - Se le conoce también como el _Anfitrión_
+        - Tiene los permisos que tendría un administrador o un _Apache committer_
+          pero es un trabajo, independientemente de los permisos. 
+        - Mantenedor/Dueño (conflicto con el rol _maintainer_ de GitHub)
+        - _Committer_ no es un término IS. Apache emplea este término para denotar
+          un rol de liderazgo tecnológico (TC++)
+        - El rol de TC se define más por sus responsabilidades que por sus
+          privilegios.
+
+- En PayPal, el debate acerca del nombre del rol supuso un reto muy serio,
+  así que el nombre importa.
+
+### Responsabilidades
+
+**Orientadas a la tecnología**
+- Mantener la fluidez del desarrollo de SW 
+- Asegurar la calidad y longevidad del SW a un nivel estratégico
+- Asegurar que las decisiones respecto a la calidad y longevidad del SW se
+  abordan/vigilan/implementan/...
+- Asumir decisiones respecto a comunidad/proceso/tecnología, aunque no sean
+  suyas.
+- Revisar y aceptar contribuciones (incluyendo asegurarse de que cumplen con
+  las líneas maestras)
+- Responsable de comunicar las líneas maestras de contribución (posiblemente
+  también de validar y establecer algunas)
```

Comment:
I don't see the term "líneas maestras" in the articles maybe is better to use "pautas", to keep it consistent